fork/dynamic-cdn-webpack-plugin/src/find.js

```diff
@@ -13,7 +13,7 @@ function findPackage(info) {
 		const {
 			packageJson: { version },
 		} = readPkgUp.sync({ cwd });
-		return version === info.version;
+		return semver.major(version) === semver.major(info.version);
```

Comment:
This is less strict as before, comparing only the major version. Wouldn't it introduce possible side effects if 2 versions are present (not deduped) and the one that is selected doesn't contain the used feature for example ?

Reply:
yes exact, it could create bugs, I have fixed by using caret syntax so now we check against semver compatible with the version it has been compiled with.